.golangci.yml

```diff
@@ -18,8 +18,8 @@ linters:
 
 linters-settings:
   depguard:
-    list-type: blacklist
-    include-go-root: false
+    list-type: denylist
+    include-go-stdlib: false
```

Comment:
per the [README](https://github.com/OpenPeeDeeP/depguard), the functionality hasn't changed, just the names.

we can still keep the old names as they keep backwards compatibility if we want.